onward/app/services/GetPopularInTagDCR.scala

```diff
@@ -0,0 +1,54 @@
+package services
+
+import com.gu.contentapi.client.utils.format.{ArticleDesign, NewsPillar, StandardDisplay}
+import common.Edition
+import contentapi.ContentApiClient
+import feed.MostReadAgent
+import layout.ContentCard
+import model.ContentFormat
+import model.dotcomrendering.{OnwardCollectionResponse, Trail}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait GetPopularInTagDCR {
```

Comment:
Same statement as above, maybe we can call this `GetPopularInTag`, and rename the other as `GetPopularInTagDeprecated`?